python/cuda_cccl/tests/parallel/examples/unique/unique_by_key.py

```diff
@@ -106,46 +106,8 @@ def compare_op(lhs, rhs):
     return h_out_keys, h_out_values, num_selected
 
 
-def keys_only_unique_example():
```

Comment:
This example was never destined to work. @NaderAlAwar could you please confirm that passing `None` for the values isn't quite supported? I couldn't find a test in C++ that did something like that.

(my fault BTW, pretty sure I added this example)

Reply:
Yeah unlike other key value algorithms this one requires values that aren't None. There is a separate [`Unique()`](https://nvidia.github.io/cccl/cub/api/structcub_1_1DeviceSelect.html#_CPPv4I000EN3cub12DeviceSelect6UniqueE11cudaError_tPvR6size_t14InputIteratorT15OutputIteratorT20NumSelectedIteratorTN4cudaSt7int64_tE12cudaStream_t) for this case (which we do not support yet).